src/services/WorkspaceWorkQueuesApi.ts

```diff
@@ -89,10 +89,4 @@ export class WorkspaceWorkQueuesApi extends WorkspaceApi {
 
     return mapper.map('FlowRunResponse', data, 'FlowRun')
   }
-
-  public async getWorkQueueStatus(id: string): Promise<WorkQueueStatus> {
-    const { data } = await this.get<WorkQueueStatusResponse>(`/${id}/status`)
-
-    return mapper.map('WorkQueueStatusResponse', data, 'WorkQueueStatus')
-  }
```

Comment:
this API is still available server-side, but it's no longer being used in the UI so I'm removing it to reduce confusion between the old work queue status and new work queue status.

today, the status comes directly off the work queue entity like `workQueue.status`